infra/recipes/docker-compose/oh-abs-spark/docker-compose.yml

```diff
@@ -0,0 +1,96 @@
+version: "3.3"
+services:
+  openhouse-tables:
+    container_name: local.openhouse-tables
+    extends:
+      file: ../common/oh-services.yml
+      service: openhouse-tables
+    volumes:
+      - ./:/var/config/
+    depends_on:
+      - openhouse-housetables
+      - prometheus
+      - opa
+    environment:
+      - AZURE_CLIENT_ID= # add your client id here
```

Comment:
Not a blocker but can we give https://hub.docker.com/r/microsoft/azure-storage-azurite a shot for spinning up a purely local docker setup?

Reply:
Yes I remember trying that briefly but then pivoting.